src/Macros/UriTranslationMacro.php

```diff
@@ -28,11 +28,11 @@ public static function register()
             // Attempt to translate each segment. If there is no translation
             // for a specific segment, then its original value will be used.
             $translations = $segments->map(function ($segment) use ($locale) {
```

Comment:
Forgot to use $namespace here (see failing tests)

Reply:
Thanks for the correction! Indeed, now it should pass all tests.